File: java/eyes.selenium.java/src/main/java/com/applitools/eyes/selenium/fluent/ISeleniumCheckTarget.java
package com.applitools.eyes.selenium.fluent;

import com.applitools.eyes.selenium.TargetPathLocator;

import java.util.List;

public interface ISeleniumCheckTarget extends IScrollRootElementContainer {
//    By getTargetSelector();
//    WebElement getTargetElement();
    TargetPathLocator getTargetPathLocator();
    List<FrameLocator> getFrameChain();
}

File: dotnet/Eyes.Selenium.DotNet/Fluent/ISeleniumCheckTarget.cs
using System.Collections.Generic;

namespace Applitools.Selenium.Fluent
{
    interface ISeleniumCheckTarget : ITargetContainer, IScrollRootElementContainer
    {
        IList<FrameLocator> GetFrameChain();

        CheckState State { get; set; }

        bool? GetUseCookies();
    }
}